nsis-download/Cargo.toml

```
@@ -1,14 +1,19 @@
 [package]
 name = "nsis-download"
 version = "0.1.0"
-edition = "2021"
-license = "MIT OR Apache-2.0"
+authors.workspace = true
+edition.workspace = true
+license.workspace = true
 
 [lib]
-crate-type = ["cdylib"]
+crate-type = ["lib", "cdylib"]
```

I think you can specify `rlib` type
```suggestion
crate-type = ["rlib", "cdylib"]
```
then re-export it in nsis-tauri
```
pub use nsis_download::*;
pub use nsis_process::*;
```

---

oh my god i want to cry. i only tested this before adding `"lib"` to crate-type 😡